test/core/GnosisSafe.GuardManager.spec.ts

```diff
@@ -10,10 +10,12 @@ import { chainId } from "../utils/encoding";
 describe("GuardManager", async () => {
 
     const [user1, user2] = waffle.provider.getWallets();
+    const guardEip165Calldata = "0x01ffc9a7e6d7a83a00000000000000000000000000000000000000000000000000000000" // Supports guard interface
```

Comment:
You can use ethersjs to generate this calldata.

Reply:
I thought it was not possible due to being an interface and not a contract

Reply:
Done

Reply:
You can use `getContractFactory` on the `hre.ethers` object (see https://hardhat.org/plugins/hardhat-deploy-ethers.html#helpers). This way you don't need to create the Contract instance and will have access to the interface (see https://docs.ethers.io/v5/api/contract/contract-factory/#ContractFactory--properties)

Reply:
I tried but I get ` NomicLabsHardhatPluginError: You are trying to create a contract factory for the contract Guard, which is abstract and can't be deployed.
If you want to call a contract using Guard as its interface use the "getContractAt" function instead.`